src/pymordemos/delay.py

```diff
@@ -28,33 +29,13 @@ def dH(s):
         return np.array([[-(tau * s + tau + 1) * np.exp(-s) / (tau * s + 1) ** 2]])
 
     tf = TransferFunction(1, 1, H, dH)
+    w = np.logspace(-1, 3, 500)
+    fom_properties(tf, w)
 
     # Transfer function IRKA (TF-IRKA)
-    tf_irka_reductor = TFIRKAReductor(tf)
-    rom = tf_irka_reductor.reduce(r, maxit=1000)
-
-    # Final interpolation points
-    sigma_list = tf_irka_reductor.sigma_list
-    fig, ax = plt.subplots()
-    ax.plot(sigma_list[-1].real, sigma_list[-1].imag, '.')
-    ax.set_title('Final interpolation points of TF-IRKA')
-    ax.set_xlabel('Re')
-    ax.set_ylabel('Im')
-    plt.show()
-
-    # Magnitude plots
-    w = np.logspace(-1, 3, 200)
-
-    fig, ax = plt.subplots()
-    tf.mag_plot(w, ax=ax)
-    rom.mag_plot(w, ax=ax, linestyle='dashed')
-    ax.set_title('Magnitude plots of the full and reduced model')
-    plt.show()
-
-    fig, ax = plt.subplots()
-    (tf - rom).mag_plot(w, ax=ax)
-    ax.set_title('Magnitude plots of the error system')
-    plt.show()
+    with warnings.catch_warnings():
+        warnings.simplefilter('ignore')  # suppress IntegrationWarning from H2 error computation
```

Comment:
I think it would be good if you elaborate a little bit more why the warning can be ignored.

Reply:
I guess it's probably wrong to say that this warning can be ignored. How about adding recommendations for better things to do? E.g., reducing the integration tolerance in `TransferFunction.h2_norm` via `quad_kwargs`, or computing the H2 error differently (in situations like these, when the H2 norm of the FOM is easy to compute, the H2 error can be computed more directly using the inner product formula based on the pole-residue form; .e.g,, see Lemma 1 in [here](https://doi.org/10.1007/978-1-4419-5757-3_1)).

Reply:
Looking more at the example, I also think we can't ignore the warning. Adding recommendations to increase `epsabs` and `limit` arguments via `quad_kwargs` seems like a good option. How would you efficiently compute poles and residues for H2-norm computation if you only have the transfer function available? 

For the `run_mor_method` maybe we can add dictionaries for parameters for `h2_norm` and `reduce` instead of `reduce_kwargs`? Or use a different benchmark in `delay.py` which results in a nicer error transfer function, so we don't have to deal with this problem at all.

Reply:
@lbalicki @sdrave I switched now the H2 error computation to use the inner product formula I mention. Let me know what you think.

Reply:
Looks a bit like this shout not be part of a demo script. Would it be possible to move the code somewhere else?

Reply:
Hm, I guess I could use `pymor.reductors.h2._lti_to_poles_b_c`. Something like an `h2_inner` method could be added to `InputOutputModel` for computing the H2 inner product, but it would only accept `LTIModels`, so it seems a bit too specific. @sdrave Do you have another idea for where to put such small functions?

Reply:
Maybe just add it to `LTIModel`?

Reply:
I now moved `h2_norm` from `TransferFunction` to `InputOutputModel` and added `h2_inner` to `InputOutputModel`.